unit-tests/shared/src/test/require-jdk11/org/scalanative/testsuite/javalib/nio/file/PathTest.scala

```diff
@@ -0,0 +1,88 @@
+package javalib.nio.file
```

Comment:
From checking existing art, StringTestOnJDK15.scala
I think the convention is to do something like:
`PathTestOnJDK11.scala` (in the new require-jdk11 directory introduced in this PR).

Seems picky but consistency pays off when one has to 
debug at 2 in the morning.